queries/cpp/textobjects.scm

```diff
@@ -55,3 +55,10 @@
 
 (new_expression
   (argument_list) @call.inner) @call.outer
+
+(declaration
+ type: (_) @assignment.lhs
```

Comment:
Doesn't this query also work for C? C++ could inherit from it?

Reply:
@amar-laksh friendly ping

Reply:
Hiya, sorry about this. Been a bit busy. It should work for assignments in C as well since the AST should be same for that part. I am going to have a look and update this by weekend?